packages/react-examples/src/react-flex/Flex/Flex.stories.scss

```diff
@@ -0,0 +1,12 @@
+.wrapper {
```

Comment:
Let's get rid of this file ? we don't want sass back again 😱😱

Reply:
What's the recommended styling solution for our stories? Couldn't find any examples.

Reply:
Yup plain old `makeStyles`